gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Jan 18 15:22:24 CET 2019
+#Thu Aug 18 15:53:42 EEST 2022
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```

Comment:
deserves changelog entry, doesn't it?

Reply:
Well, this is gradle used to build the project, but I'll drop a comment about other dependency updates